crates/forge/tests/it/zk/paymaster.rs

```diff
@@ -0,0 +1,35 @@
+//! Forge tests for zksync contracts.
+
+use foundry_config::fs_permissions::PathPermission;
+use foundry_test_utils::util;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_zk_contract_paymaster() {
+    let (prj, mut cmd) = util::setup_forge(
+        "test_zk_contract_paymaster",
+        foundry_test_utils::foundry_compilers::PathStyle::Dapptools,
+    );
+    util::initialize(prj.root());
+
+    cmd.args([
+        "install",
+        "OpenZeppelin/openzeppelin-contracts",
+        "cyfrin/zksync-contracts",
```

Comment:
I'd avoid relying on 3rd party if we can use 1st party https://github.com/matter-labs/era-contracts